packages/AutoTDD-Core.package/ATDDGitHubAction.class/class/defaultRefreshInterval.st

```diff
@@ -1,4 +1,4 @@
 constants
 defaultRefreshInterval
 
-	^ (Time hour: 0 minute: 0 second: 20) asSeconds
\ No newline at end of file
+	^ (Time hour: 0 minute: 4 second: 20) asSeconds
```

Comment:
Even though I like the idea, maybe we should pick something more serious considering that it will be graded X)

```suggestion
	^ (Time hour: 0 minute: 5 second: 0) asSeconds
```